src/features/import/utils/prepareImportOperations.spec.ts

```diff
@@ -267,28 +267,28 @@ describe('prepareImportOperations()', () => {
       const result = prepareImportOperations(configData);
       expect(result).toEqual([
         {
-          fields: {
+          data: {
             ext_id: '123',
             first_name: 'Jane',
             last_name: 'Doe',
           },
           op: 'person.import',
-          organizations: [272],
+          organization: [272],
```

Comment:
As commented elsewhere, this property should not be renamed. It should remain `organizations` which is what the backend expects.